kratos/geometries/prism_3d_15.h

```diff
@@ -838,10 +838,17 @@ class Prism3D15
         const CoordinatesArrayType& rCoordinates
         ) const override
     {
+        // Ensure size is correct
+        if (rResult.size() != 15) {
+            rResult.resize(15, false);
+        }
```

Comment:
the check is done internally as well
```suggestion
        rResult.resize(15, false);
```